orca-redis/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java

```diff
@@ -55,6 +56,7 @@
 import static redis.clients.jedis.BinaryClient.LIST_POSITION.BEFORE;
 
 @Component
+@ConditionalOnProperty(value = "executionRepository.redis.enabled", matchIfMissing = true)
```

Comment:
So that's on by default?

Reply:
Yes. This is backwards compatible with existing configurations.